packages/menu/src/MenuItem.ts

```diff
@@ -460,13 +472,20 @@ export class MenuItem extends LikeAnchor(
         super.firstUpdated(changes);
         this.setAttribute('tabindex', '-1');
         this.addEventListener('keydown', this.handleKeydown);
+        this.addEventListener('mouseover', this.handleMouseover);
         this.addEventListener('pointerdown', this.handlePointerdown);
         this.addEventListener('pointerenter', this.closeOverlaysForRoot);
         if (!this.hasAttribute('id')) {
             this.id = `sp-menu-item-${randomID()}`;
         }
     }
-
+    handleMouseover(event: MouseEvent): void {
+        const target = event.target as HTMLElement;
+        if (target === this) {
+            this.focus();
+            this.focused = false;
```

Comment:
Can you clarify this pattern for me? We run the focus to set focus on the menu item but then why is the local focused attribute being set to false?